### PR TITLE
Make the resolve route return BadRequest when the supplied package

### PR DESCRIPTION
**Pull request by stevana**
id doesn't exist. Found by @tkfu.

### CODE REVIEW

File: external-resolver/src/main/scala/org/genivi/sota/resolver/Boot.scala [hunk 1]
```diff
@@ -59,16 +59,8 @@ class Routing(db: Database)
...
-      (get & refinedPackageId) { (name, version) =>
-        complete(db.run(Resolve.resolve(name, version)))
-      }
-    }
-  }
-
   def packageFiltersHandler: ExceptionHandler = ExceptionHandler {
-    case err: PackageFilters.MissingPackageException =>
+    case err: Packages.MissingPackageException.type =>
       complete(StatusCodes.BadRequest ->
```
Comment (koshelev) [inline external-resolver/src/main/scala/org/genivi/sota/resolver/Boot.scala]: It should be NotFound, not BadRequest

**Comment by stevana** [comment]
Ready for review.

**Comment by stevana** [comment]
Ready!

**Comment by koshelev** [comment]
The should be only one! (commit)

**Comment by stevana** [comment]
Mater of taste. They are two separate issues.

**Comment by koshelev** [comment]
In this case it should be two PRs :)

**Pull request merged**